doc/reference/reference_lua/box_info/synchro.rst

```diff
@@ -0,0 +1,142 @@
+..  _box_info_synchro:
+
+================================================================================
+box.info.synchro
+================================================================================
+
+..  module:: box.info
+
+..  data:: synchro
+
+    Since version :doc:`2.8.1 </release/2.8.1>`.
+    Show the current state of synchronous replication.
+
+    In :ref:`synchronous replication <repl_sync>`, a transaction is considered committed only after achieving
+    the required quorum number.
+    While the transactions are collecting confirmations from remote nodes, these transactions are waiting in the queue.
+
+    The following information is provided:
+
+    *   ``queue``:
+
+        -   ``owner`` -- ID of the replica that owns the synchronous transaction queue.
+            Once an owner instance appears, all other instances become read-only.
+            If the ``owner`` field is ``0``, then every instance is writeable,
```

Comment:
if owner is 0, then every instance **may** be writeable.

Reply:
Fixed